tests/test_universe.py

```diff
@@ -168,16 +168,16 @@ def _check(t, values):
 class Point3dTB(ModMeshTB):
 
     def test_construct(self):
-        Vector = self.kls
+        Point = self.Point
 
         # Construct using positional arguments
-        vec = Vector(1, 2, 3)
+        vec = Point(1, 2, 3)
```

Comment:
You may replace `vec` with `pnt` if you want, but leaving it as `vec` is fine too.